### PR TITLE
Add doc for frontend development

### PR DESCRIPTION
**Pull request by davisshaver**

### CODE REVIEW

**Comment by davisshaver** [comment]
@lisaej, review/merge?

File: wordpress-development/frontend-development.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#Frontend Development
+
+We use [Grunt](http://gruntjs.com/getting-started) as our build sytem for compiling SASS and Javascript, and running various tests/linting functions.
+
+1. Make sure you have [Grunt](http://gruntjs.com/getting-started) installed globally.
+5. Go to your theme directory and install the necessary libraries using <code>npm install</code>
+6. Vola! Run <code>grunt watch</code> in your terminal and leave it there. It will watch your change and recompile everything for you. You should see something like this.
```
Comment (danielbachhuber) [inline wordpress-development/frontend-development.md]: These numbers seem off

File: wordpress-development/frontend-development.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#Frontend Development
```
Comment (danielbachhuber) [inline wordpress-development/frontend-development.md]: Spacing

File: wordpress-development/frontend-development.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Frontend Development
+
+We use [Grunt](http://gruntjs.com/getting-started) as our build sytem for compiling SASS and Javascript, and running various tests/linting functions.
+
+1. Make sure you have [Grunt](http://gruntjs.com/getting-started) installed globally.
+1. Go to your theme directory and install the necessary libraries using <code>npm install</code>
```
Comment (lisaej) [inline wordpress-development/frontend-development.md]: Numbering. And a small typo.(Vola)

Comment (davisshaver) [inline wordpress-development/frontend-development.md]: Using `1.` for every element in OL is alright with markdown according to Gruber Tha God

**Comment by davisshaver** [comment]
@lisaej made a couple changes. Can you merge?

**Pull request merged**